src/components/inputs/_input.scss

```diff
@@ -193,6 +192,24 @@
   align-items: baseline;
 }
 
+.flex_row {
```

Comment:
Minor but i think this should be `camelCase` like the others for consistency

Reply:
Nice catch